.golangci.yml

```diff
@@ -57,27 +31,53 @@ linters:
     - paralleltest
     - prealloc
     - promlinter
-    - staticcheck
-    - tenv
     - testifylint
     - thelper
     - tparallel
-    - typecheck
     - unconvert
     - unparam
-    - unused
     - usestdlibvars
     - wastedassign
     - whitespace
     - wrapcheck
-linters-settings:
-  copyloopvar:
-    check-alias: true
-  goconst:
-    ignore-strings: "true"
-  gocritic:
-    disabled-checks:
-      - assignOp  # Keep it disabled for readability
-      - ifElseChain
-  testifylint:
-    enable-all: true
+  settings:
+    copyloopvar:
+      check-alias: true
+    goconst:
+      ignore-strings: "true"
+    gocritic:
+      disabled-checks:
+        - assignOp
+        - ifElseChain
+    testifylint:
+      enable-all: true
+  exclusions:
+    generated: lax
```

Comment:
Maybe go ahead and drop this? Dropped it in Argo CD and didn't hit any additional issues.